src/Compilers/VisualBasic/vbc2/vbc2.vcxproj.filters

```diff
@@ -34,5 +34,6 @@
       <Filter>Resource Files</Filter>
     </ResourceCompile>
     <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
+    <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
```

Comment:
@gafter: This seems to be an accidental additional by the VS tooling. @mattscheffer is looking into removing it.